.github/workflows/deploy-cloudrun-it.yml

```diff
@@ -62,6 +62,62 @@ jobs:
         ENV: TEST_ENV=TEST_VAR,TEST_ENV2=TEST_VAR2
         COUNT: 2
 
+  secret-manager:
+    name: with Secret Manager
+    if: ${{ github.event_name == 'push' || github.repository == github.event.pull_request.head.repo.full_name }}
+    runs-on: ubuntu-latest
+    steps:
+    - id: service
+      run: echo ::set-output name=service::run-secret-manager-$GITHUB_SHA
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@master
+      with:
+        node-version: 12.x
+    - name: Build Javascript
+      run: |-
+        npm install
+        npm run build
+
+    - name: Create Service with Secret Manager
+      id: deploy_1
+      uses: ./
+      with:
+        credentials: ${{ secrets.DEPLOY_CLOUDRUN_SA_KEY_JSON }}
+        image: gcr.io/cloudrun/hello
+        service: ${{ steps.service.outputs.service }}
+        secrets: MY_SECRET=secret_value:latest
+
+    - name: Setup Authentication with gcloud
+      uses: google-github-actions/setup-gcloud@master
+      with:
+        service_account_key: ${{ secrets.DEPLOY_CLOUDRUN_SA_KEY_B64 }}
+        export_default_credentials: true
+
+    - name: Integration Tests
+      run: npm run e2e-tests
+      env:
+        URL: ${{ steps.deploy_1.outputs.url }}
+        SERVICE: ${{ steps.service.outputs.service }}
+        SECRET_ENV: MY_SECRET=secret_value:latest
+
+    - name: Update service with mounted secret
+      id: deploy_2
+      uses: ./
+      with:
+        credentials: ${{ secrets.DEPLOY_CLOUDRUN_SA_KEY_JSON }}
+        image: gcr.io/cloudrun/hello
+        service: ${{ steps.service.outputs.service }}
```

Comment:
Can you delete this service in the Clean up job?

Reply:
Done in baab803480151199bf6418b270fd8cc4162b1bbe